### PR TITLE
Added README, templates for configs

### PR DESCRIPTION
**Pull request by VadVergasov**
Added templates for configs that contains necessary variables.
Added README with installation guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Before running server install all requirements:
+
+    python3 -m venv clist_environment
+    source env/bin/activate
+    pip install -r requirements/requirements.txt
+    sudo ./requirements/packages.sh
+
+Now rename configs by:
+
+    mv pyclist/conf.py.template pylcist/conf.py
```
Comment (aropan) [inline README.md]: Use `cp` instead `mv`.

File: pyclist/conf.py.template [hunk 2]
```diff
@@ -0,0 +1,10 @@
+EMAIL_HOST_PASSWORD = '{str}'
+SECRET_KEY = '{str}'
+TELEGRAM_TOKEN = '{int}:{str}'
+TELEGRAM_ADMIN_CHAT_ID = {int}
+
+DB_HOST = '{str}'
+DB_PORT = '{int}'
+DB_NAME = '{str}'
+DB_USER = '{str}'
+DB_PASSWORD = '{str}'
```
Comment (aropan) [inline pyclist/conf.py.template]: Add new line to end of file.

File: README.md (hunk 1 above)
Comment (VadVergasov) [reply README.md]: Replaced.

File: pyclist/conf.py.template (hunk 2 above)
Comment (VadVergasov) [reply pyclist/conf.py.template]: Added line ending.

File: .pgpass.template [hunk 3]
```diff
@@ -0,0 +1 @@
+host:port:database:username:password
```
Comment (aropan) [inline .pgpass.template]: I rewrote `./requirements/configure.sh`. Remove `.pgpass.template`.

Comment (VadVergasov) [reply .pgpass.template]: Removed .pgpass.template

File: .gitignore [hunk 4]
```diff
@@ -2,3 +2,4 @@ certificate
 cache
 uwsgi.sock
 nginx/static/
+clist_environment
```
Comment (aropan) [inline .gitignore]: Rename to without `clist_`.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+Using Clist
+======
+
+First run
+------
+
+Before running server install all requirements:
+
+    python3 -m venv clist_environment
+    source env/bin/activate
```
Comment (aropan) [inline README.md]: And:
```
python3 -m venv environment
source environment/bin/activate
```

File: .gitignore (hunk 4 above)
Comment (aropan) [reply .gitignore]: Or simple `env`.

File: README.md (hunk 5 above)
Comment (VadVergasov) [reply README.md]: Renamed to `environment`.

**Pull request merged**